taxi/urls.py

```diff
@@ -7,16 +7,40 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    CarCreateView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    CarUpdateView,
+    ManufacturerDeleteView,
+    CarDeleteView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
-        name="manufacturer-list",
+        name="manufacturer-list"
+    ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
```

Comment:
Consider adding a trailing slash to the 'manufacturers/<int:pk>/update' path for consistency with other paths, such as 'cars/<int:pk>/update/'.